### ISSUE TITLE
miniature_golf has ame colour as camp_site

### ISSUE THREAD

**Issue by BertMule** [gravitystorm/openstreetmap-carto#4919]
A miniature_golf has the same colour as a camp_site.
With the first as part of the latter, there is no contrast. 
Which beats the intend of separately modelling them.

As shown [here](https://www.openstreetmap.org/#map=19/50.48046/5.58857),

**Comment by imagico** [comment]
Yes, this goes back to the decision from #4381, which was prepared in #4212 - where the choice of color was discussed in more depth.

Currently we use the same color for:

* `tourism=camp_site`
* `tourism=caravan_site`
* `tourism=picnic_site`
* `leisure=golf_course`
* `leisure=miniature_golf`

That is not ideal - but the problem is mitigated by the fact that all of these are rendered with a point symbol as well - resolving the ambivalence in color.

As discussed in #4212 already, solving this overloading of color use is difficult given the large number of green colors we already use.  If anyone would like to look into this, disentangling the much broader overloading of the very light blue-green color (labeled `@leisure`) would most certainly need to be the first step - see https://github.com/gravitystorm/openstreetmap-carto/pull/4212#issuecomment-707680821

**Comment by dch0ph** [comment]
Yes, a new green is unlikely to work.

There similarly wasn't much enthusiasm for highlighting borders of `leisure=golf_course` (which would be an alternative): #4870
Not least because there would normally be some kind of barrier (or natural feature) demarcating a golf course.

Wouldn't most `leisure=miniature_golf` similarly have a barrier? In this case, simply more detailed mapping would make the issue go away.

An alternative would be to render `leisure=miniature_golf` as `leisure=playground` on the basis that crazy golf arguably has more similarities to a constructed playground than a "semi-natural" golf course. `leisure=playground` has a thin border would help with demarcation if a barrier is not mapped.

**Comment by BertMule** [comment]
I would be in favour of at least some border.

A distinctive colour would be better.
But I understand the problem there.

Comparable with outdoor seatings #4792.
